apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -39,60 +26,88 @@ type Story = StoryObj<typeof Tooltip>;
 
 export const TopPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.TOP,
+    placement: TooltipPlacement.Top,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
```

Comment:
I changed each of the `trigger` and `children` in each story to be in `<span>` tags rather than `<>` tags. The reason is because when you look at the example code generated in storybook, you see some React implementation details there when you use `<>`.

We want to optimise for really nice auto-generated code examples on the "Docs" pages. Therefore I switched all of these to `<span>`